domenetjenester/registerinnhenting/src/main/java/no/nav/foreldrepenger/domene/registerinnhenting/impl/startpunkt/StartpunktUtlederPersonopplysning.java

```diff
@@ -70,8 +71,14 @@ private List<StartpunktType> hentAlleStartpunktForPersonopplysninger(BehandlingR
             startpunkter.add(StartpunktType.UTTAKSVILKÅR);
         }
         if (poDiff.erBarnDødsdatoEndret()) {
-            FellesStartpunktUtlederLogger.loggEndringSomFørteTilStartpunkt(this.getClass().getSimpleName(), StartpunktType.BEREGNING, "barnets dødsdato", grunnlag1.getId(), grunnlag2.getId());
-            startpunkter.add(StartpunktType.BEREGNING);
+            //TODO bedre måte??
+            if (ref.fagsakYtelseType() == FagsakYtelseType.FORELDREPENGER) {
```

Comment:
@pekern @jolarsen @espenwaaga Hva tror dere er best?

1. Startpunktutleder må sjekke ytelsetype, fp -> dekningsgrad, svp/es -> beregning
2. Sette DEKNINGSGRAD som startpunkttype for alle ytelser, men ha tomme dekningsgrad-steg for svp og es
3. Legge dekningsgrad steget midt midt i beregning og fortstte å bruke startpunkttype BEREGNING ved dødsfall/annen part endret dekningsgrad?

Reply:
Evt andre løsninger?

Reply:
Siden dekningsgrad kun gjelder FP - i likhet med besteberegning - og først brukes i beregning - alternativ 3?
Spørsmålet er hvor tidlig/sent i beregning man faktisk bruker DG - FORESLÅ_BG ?

Reply:
StartpunktTjenste for ES bryr seg bare om IV-startpunkt (ellers ville den gamle koden feilet ....)
Den enkle tilnærmingen er å legge Dekningsgrad inne blant  beregningstegene (slik som besteberegn)